spec/features/visitors/password_recovery_spec.rb

```diff
@@ -40,11 +40,8 @@ def reset_password_and_sign_back_in(user, password = 'a really long password')
       user = create(:user, :unconfirmed)
       confirm_last_user
       reset_email
-      visit new_user_password_path
-      fill_in 'Email', with: user.email
-      click_button t('forms.buttons.continue')
-      open_last_email
-      click_email_link_matching(/reset_password_token/)
+
+      trigger_reset_password_and_click_email_link(user.email)
```

Comment:
I have tended to put supporting methods as near to where they are used as possible, mostly just for my own convenience. No strong opinion. I can move them all to the same place.